projects/plugins/social/src/js/components/icon-text/styles.module.scss

```diff
@@ -0,0 +1,62 @@
+.column {
+	grid-column: 1 / -1;
+	display: grid;
+	grid-template-columns: auto 1fr;
+	grid-template-rows: repeat( 2, auto );
+	column-gap: calc( var( --spacing-base ) * 3 );
+	row-gap: calc( var( --spacing-base ) * 2 );
+
+	> * + * {
+		margin-top: calc( var( --spacing-base ) * 2 );
+	}
+	
+	@supports ( display: grid ) {
+		> * + * {
+			margin-top: 0;
+		}
+	}
```

Comment:
I only just thought about this (and don't feel the need to change it), but this is also a good use case for CSS variables:

```scss
.column {
	--item-spacing: calc( var( --spacing-base ) * 2 );

	> * + * {
		margin-top: var( --item-spacing );
	}

	@supports ( display: grid ) {
		--item-spacing: 0;
	}
}
```